arduino-ide-extension/src/browser/serial/monitor/monitor-widget.tsx

```diff
@@ -102,6 +106,11 @@ export class MonitorWidget extends ReactWidget {
     this.clearOutputEmitter.fire(undefined);
     this.update();
   }
+  
+  copyOutput(): void {
+    this.copyOutputEmitter.fire();
+    this.update();
+  }
```

Comment:
```suggestion

  copyOutput(): void {
    this.copyOutputEmitter.fire();
  }
```

It's not needed to update the widget after copying the text; no need to rerender.

Reply:
Resolved in [1a95c33](https://github.com/arduino/arduino-ide/pull/2718/commits/1a95c3358a9be8f7fdfdbacfd34dcbe838641c30)